nservicebus/recoverability/index_retrycount_core_[6.0,).partial.md

```diff
@@ -20,7 +20,7 @@ Given a variety of immediate and delayed configuration values here are the resul
 | 1                | 2              | 6                       |
 | 2                | 2              | 9                       |
 | 1                | 3              | 8                       |
-| **3**            | **5**          | **24  (default)**       |
+| **5**            | **3**          | **24  (default)**       |
```

Comment:
What is the reason for switching these values?

Reply:
It looks like these were just in the wrong columns - ImmediateRetries is the first column, and DelayedRetries is the second column.

Elsewhere in the page it [mentions the default for Immediate Retries is 5](https://docs.particular.net/nservicebus/recoverability/#immediate-retries):

> By default up to five immediate retries are performed if the message processing results in exception being thrown.

Implies the [default for Delayed Retries is 3](https://docs.particular.net/nservicebus/recoverability/#delayed-retries):

> Delayed Retries introduces another level of retry mechanism for messages that fail processing. Delayed Retries schedules message delivery to the endpoint's input queue with increasing delay, by default first with 10 seconds delay, then 20, and lastly with 30 seconds delay.

And then [states 5 for immediate and 3 for delayed](https://docs.particular.net/nservicebus/recoverability/#total-number-of-possible-retries-scale-out-multiplier):

> When taking the default values for immediate and delayed retries (five and three, respectively) and 6 instances the total number of attempts will be a minumum of (5+1)(3+1)=24 attempts and a maximum of 246=144 attempts.

Reply:
👍 thanks for the explanation @matthewhermann, that makes sense.

Reply:
Incidentally, the `*` characters in that last sentence are being interpreted as italics markdown. I've fixed that in https://github.com/Particular/docs.particular.net/pull/5013

Reply:
Oh yes, I missed that - nice one 👍